packages/interface/src/components/admin-menu-toggle/style.scss

```diff
@@ -10,7 +10,6 @@
 		border: none;
 		border-radius: 0;
 		height: auto;
-		margin-right: -8px;
```

Comment:
Without this the logo and the Inserter seem too far apart.

Reply:
Here's what I see when I look at your suggested spacing:

<img width="679" alt="image" src="https://user-images.githubusercontent.com/191598/80131902-ee571980-8568-11ea-9d40-3fee3347ef1a.png">

All the green spaces are the same size, but then there's that random looking red space.

What I'd expect is for the space around the + button to be the same on all sides.

<img width="596" alt="image" src="https://user-images.githubusercontent.com/191598/80132117-37a76900-8569-11ea-852d-d5cc9412ab13.png">

Reply:
This is the grid that G2 was designed towards:

<img width="1358" alt="timline before" src="https://user-images.githubusercontent.com/1204802/80192432-74ca3480-8617-11ea-992c-61c4ec122b51.png">

But then there's a toggled state:

<img width="798" alt="Screenshot 2020-04-24 at 10 39 57" src="https://user-images.githubusercontent.com/1204802/80192758-f7eb8a80-8617-11ea-9ec5-049c7ca9c847.png">

That toggled state is 32px wide, which is optically balanced to be between the 24px button and the outer edge of the block toolbar. If the black square was 36px, then sequential icons, bold italic and so on, would but up against each other.

It's an ongoing effort to make everything adhere properly to the grid, and actually that 32px size is one of the things that I promised Riad that we should look into. As he noted, most buttons are 36px in dimensions, which feels too big for the inserter and certainly is too big for toggle buttons in the block toolbar. It also makes the focus style for the inserter inconsistent with the other buttons. So there's definitely an inconsistency here that I think we should address holistically.

In the mean time, though, I would consider this PR less of a design discussion and more of a fix for a regression given the discussion to add a left padding to match the G2 mockups here: https://github.com/WordPress/gutenberg/pull/20685#issuecomment-596215638

Would you mind if we merged this fix and then opened a separate ticket to discuss this margin? We could make it a ticket about how to apply that grid to the top toolbar, dimensions of the buttons, spacing between, all of that, and then address it all in one fell swoop!

Reply:
Don’t let my complaining stop you from merging.

Reply:
Alright, I created #21870 as a good and visible place to discuss this!

Reply:
I just think all buttons should have the same size no matter where they are 🤷‍♂️